src/service/topics.ts

```diff
@@ -172,6 +172,19 @@ const inviteCommunityMember = async (topicId, memberIds) => {
   }
 };
 
+const putCommunityImage = async (topicName, icon, cover) => {
+  try {
+    const resApi = await api.put(`/topics/${topicName}`, {
+      icon,
+      cover
+    });
+    return resApi.data;
+  } catch (error) {
+    crashlytics().recordError(new Error(error));
+    return error.response.data;
+  }
+};
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `putCommunityImage` function does not handle errors properly. It logs the error with Crashlytics, which is good for tracking issues, but it also returns the error response data. This could potentially leak sensitive information to the caller of this function. Instead, consider returning a generic error message or an error object that doesn't contain sensitive data.

```diff
    crashlytics().recordError(new Error(error));
-184:     return error.response.data;
+184:     return { success: false, message: 'Failed to update community image' };
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->